GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -136,6 +136,9 @@ public sealed partial class FormCommit : GitModuleForm
             + Environment.NewLine + "Use this feature when a file is big and never change."
             + Environment.NewLine + "Git will never check if the file has changed that will improve status check performance.");
         private readonly TranslationString _stopTrackingFail = new TranslationString("Fail to stop tracking the file '{0}'.");
+
+        private readonly TranslationString _statusBarBranchWithRemote = new TranslationString("{0} -> {1}/{2}");
```

Comment:
This string don't need to be translated.
I have the [changes](https://github.com/pmiossec/gitextensions/commit/5f0b5444efb4e4e92e72837456d8fa9258aa0787) in [my branch](https://github.com/pmiossec/gitextensions/tree/show-remote-branch).

You could integrate it if that suits you.